infrastructure/main.tf

```
@@ -22,9 +22,39 @@ module "key-vault" {
   create_managed_identity = true
 }
 
+locals {
+  vaultName = "${var.raw_product}-${var.env}"
+}
+
+data "azurerm_key_vault" "damages_key_vault" {
+  name = "${local.vaultName}"
```

you don't need string interpolation if you aren't interpolating
```suggestion
  name = local.vaultName
```